doc/usage.md

```diff
@@ -573,14 +573,19 @@ error result.
 
 ## How to Use Compliance Operator with HyperShift Management Cluster
 
-Compliance Operator is able to run a platform scan on the HyperShift Managment cluster
-for the Hosted Cluster with a tailoredProfile.
+[Hypershift](https://hypershift-docs.netlify.app/) allows one to create and manage clusters on existing infrastructure.
+Compliance Operator is able to create a platform scan on the [HyperShift Management Cluster](https://hypershift-docs.netlify.app/reference/concepts-and-personas/)
+for the Hosted Cluster with a TailoredProfile.
```

Comment:
Similar to my comments on another review. What are our thoughts on using literals for formal CRD names?

Reply:
Should be fixed as well.